src/openvic-simulation/country/CountryInstance.cpp

```diff
@@ -280,6 +285,11 @@ bool CountryInstance::add_reform(Reform const& new_reform) {
 	}
 }
 
+bool CountryInstance::set_value_for_slider(slider_value_t& slider, const int32_t new_value) {
+	slider.set_value(new_value);
+	return true;
+}
```

Comment:
```suggestion
void CountryInstance::set_strata_tax_rate(Strata const& strata, const tax_slider_value_t::int_type new_value) {
	tax_rate_by_strata[strata].set_value(new_value);
}
```
- This shouldn't take a `slider_value_t&`, no one outside this struct should be able to get a non-const reference to the tax rate sliders.
- Instead it should take a `Strata const&` which will be used to look up the correct slider internally.
- It doesn't need to return a `bool` if it's always going to return true.
- `new_value`'s type should match the argument for `set_value`, so `tax_slider_value_t::int_type`.
- Make sure to change the function's header declaration to:
```cpp
		void set_strata_tax_rate(Strata const& strata, const tax_slider_value_t::int_type new_value);
```